config/desktop/buster/environments/xfce/config_base/packages

```diff
@@ -36,6 +36,7 @@ fonts-ubuntu-font-family-console
 foomatic-db-compressed-ppds
 gdebi
 ghostscript-x
+Gist
```

Comment:
caps

Reply:
this got put in buster and not suppost I will refactor and re commit